Dockerfile

```diff
@@ -10,5 +10,13 @@ FROM mcr.microsoft.com/azure-functions/dotnet:2.0
 ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
     AzureFunctionsJobHost__Logging__Console__IsEnabled=true
 
-RUN apt-get update && apt-get install -y texlive-latex-base
+RUN apt-get update && apt-get install -y texlive-xetex wget unzip fonts-noto
+RUN mkdir -p /opt/fontindexer
+RUN wget https://github.com/WycliffeAssociates/FontIndexer/releases/download/1.0.1/linux-x64.zip -O /opt/fontindexer/linux-x64.zip
+RUN unzip /opt/fontindexer/linux-x64.zip -d /opt/fontindexer/
+RUN chmod u+x /opt/fontindexer/linux-x64/FontIndexer
+RUN mkdir -p /tmp/wa/fonts
+RUN cp /usr/share/fonts/truetype/noto/*-Regular.ttf /tmp/wa/fonts
+RUN mkdir -p /home/site/wwwroot
+RUN /opt/fontindexer/linux-x64/FontIndexer --source=/tmp/wa/fonts --output=/home/site/wwwroot/fonts.json

```

Comment:
These consecutive RUN statements are fine for development, but they create a lot of image layers.  For production I recommend you concatenate them into a single RUN statement, like you did on line 3.

Reply:
Good idea